manifest.yaml

```diff
@@ -0,0 +1,3 @@
+version: 0.6.0
```

Comment:
I think it mostly highlights an annoying thing I don't like: the need to update version numbers in two places. Not going to fix that for now, and thinking it through, having a manifest in the repo is useful for documentation, so we'll keep it